src/views/Home.vue

```diff
@@ -218,11 +218,12 @@ export default {
       return items;
     },
     randomSort(items) {
-      for (let i = items.length - 1; i > 0; i--) {
+      const newItems = items;
```

Comment:
This is not a copy. It's a copy of the reference, so any mutations here will also show up in the original array.

Reply:
Weird, since it does work. If I look in the Vue devtools, `this.albums` and `this.randomAlbums` are ordered differently

Reply:
Maybe it's a dev mode vs prod mode issue?

Reply:
```
> const a = [1, 2]
undefined
> const b = a;
undefined
> [b[0], b[1]] = [b[1], b[0]]
[ 2, 1 ]
> a
[ 2, 1 ]
```
This is essentially what this code does, as you can see, it doesn't work.

Reply:
I could see the issue before in dev mode. Anyway, I changed the code